src/Keycloak.AuthServices.Sdk/HttpMiddleware/AccessTokenPropagationHandler.cs

```
@@ -32,8 +32,31 @@ public AccessTokenPropagationHandler(IHttpContextAccessor contextAccessor)
         }
 
         var httpContext = this.contextAccessor.HttpContext;
-        var token = await httpContext
-            .GetTokenAsync(JwtBearerDefaults.AuthenticationScheme, "access_token");
+
```

When attempting to perform actions on behalf of a user in blazor server, the "AuthenticationScheme" does not return the access token. It is available by simply requesting "access_token" without the scheme. 
This attempts to obtain the access token in the secondary fashion if the first attempt does not succeed.
If this second attempt fails then its exception is ignored and the original exception is thrown in order to ensure the existing behaviour is still exhibited and users won't be affected.